subsys/bluetooth/controller/ll_sw/ull_conn_iso.c

```diff
@@ -723,7 +765,10 @@ void ull_conn_iso_start(struct ll_conn *acl, uint32_t ticks_at_expire, uint16_t
 	cig->cig_ref_point += EVENT_OVERHEAD_START_US;
 	cig->cig_ref_point += acl_to_cig_ref_point;
 
-	if (IS_PERIPHERAL(cig)) {
+	if (false) {
+
+#if defined(CONFIG_BT_CTLR_PERIPHERAL_ISO)
```

Comment:
`IS_PERIPHERAL` uses `IS_ENABLED(CONFIG_BT_CTLR_PERIPHERAL_ISO)` so is this necessary?